setup.py

```diff
@@ -12,6 +13,17 @@
 
 exec(compile(version_file_contents, "pytools/version.py", "exec"), ver_dic)
 
+version = sys.version_info
+requirements = [
+          "decorator>=3.2.0",
+          "appdirs>=1.4.0",
+          "six>=1.8.0",
+          "numpy>=1.6.0",
+          ]
+
+if version[0] == 3 and version[1] == 6:
+    requirements.append("dataclasses>=0.7")
+
 setup(name="pytools",
       version=ver_dic["VERSION_TEXT"],
```

Comment:
Pytools needs a version bump, and loopy/pytato should declare a dependency on that version.

Reply:
To (2020, 4, 2) or (2020, 5) or (2020, 10)?

Reply:
(2020, 4, 2)